### PR TITLE
fix(font): change calcite-font-bold to be semi-bold

### PR DESCRIPTION
**Pull request by alisonailea**
**Related Issue:** #9524

## Summary

Reassign font > font-weight > bold in semantic tokens to {core.font.weight.demi} to align with Calcite UIKit

### CODE REVIEW

**Comment by jcfranco** [comment]
@alisonailea Can you update the spec test?

File: packages/calcite-components/.stylelintrc.cjs [hunk 1]
```diff
@@ -1,13 +1,7 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [
```
Comment (driskull) [inline packages/calcite-components/.stylelintrc.cjs:4]: Can you revert this one?

**Review by driskull — APPROVED** [review]
👍

File: packages/calcite-components/.stylelintrc.cjs (hunk 1 above)
Comment (alisonailea) [reply packages/calcite-components/.stylelintrc.cjs]: thanks for the catch

**Pull request merged**